packages/components/src/components/drawer/drawer.stories.ts

```diff
@@ -36,378 +39,87 @@ export default {
   parameters: {
     ...parameters,
     controls: { exclude: ['contained'] }
-  }
-};
-
-/**
- * Default: This shows sd-drawer in its default state.
- */
-export const Default = {
-  render: (args: any) => {
-    return html` <div style="width: auto; height: 95vh; position: relative;">
-      ${generateTemplate({
-        args
-      })}
-    </div>`;
-  }
-};
-
-/**
- * Drawer with a tertiary button in the header. In case there's a requirement not to have left-padding in the header, as it's the case with the tertiary button in the `header` slot, you can override the `sd-drawer::part(header)` selector by applying a `left-padding: 0;` style.
- */
-export const ButtonInHeader = {
-  name: 'Button in Header',
-  parameters: {
-    controls: { exclude: ['header', 'contained'] }
   },
-  render: (args: any) => {
-    return html` <div class="header-with-btn" style="width: auto; height: 95vh; position: relative;">
+  decorators: [
+    (story: any) => html`
       <style>
-        .header-with-btn sd-drawer::part(header) {
-          padding-left: 0;
+        sd-drawer::part(overlay) {
+          background-color: #051530;
+          opacity: 0.9;
         }
       </style>
-      ${generateTemplate({
-        constants: [
-          {
-            type: 'slot',
-            name: 'header',
-            value: `<sd-button slot='header' variant='tertiary'>Header<sd-icon library="global-resources" name="system/arrow-left" slot="icon-left"></sd-icon></sd-button>`
-          }
-        ],
-        args
-      })}
-    </div>`;
-  }
+      ${story()}
+    `
+  ] as unknown
 };
 
 /**
- * If you do not need a header, you can remove it by applying the `no-header` attribute.
+ * Default: This shows sd-drawer in its default state.
  */
-export const NoHeader = {
-  name: 'No Header',
-  parameters: {
-    controls: { exclude: ['no-header', 'header', 'contained'] }
-  },
+export const Default = {
+  name: 'Default',
   render: (args: any) => {
-    return html` <div style="width: auto; height: 95vh; position: relative;">
+    return html` <div style="width: auto; height: 40vh; position: relative;">
       ${generateTemplate({
-        constants: [
-          {
-            type: 'attribute',
-            name: 'no-header',
-            value: true
-          }
-        ],
         args
       })}
     </div>`;
   }
 };
 
 /**
- * If you do not need a footer, you can remove it by omitting the footer slot.
+ * Use the `open` attribute to set the state of the drawer to open.
  */
-export const NoFooter = {
-  name: 'No Footer',
-  parameters: {
-    controls: { exclude: ['footer', 'contained'] }
-  },
-  render: (args: any) => {
-    return html` <div style="width: auto; height: 95vh; position: relative;">
-      ${generateTemplate({
-        constants: [
-          {
-            type: 'slot',
-            name: 'footer',
-            value: ''
-          }
-        ],
-        args
-      })}
-    </div>`;
-  }
-};
+export const Open = {
+  name: 'Open',
```

Comment:
We said we would like to show this for open:

![image](https://github.com/user-attachments/assets/3fc71302-12de-450c-b3cc-8a912271d95a)

Reply:
Updated